### PR TITLE
fix(debug): fix debug gui tools.

### PR DESCRIPTION
**Pull request by gchoqueux**
- fix charts div position;
- fix debug event;
- fix position camera debug;
- refactor style charts;

tools gui

### CODE REVIEW

File: test/functional/GlobeControls.js [hunk 1]
```diff
@@ -26,6 +26,8 @@ describe('GlobeControls with globe example', function _() {
 
                     return target;
                 };
+
+            menuGlobe.gui.remove(cRL);
```
Comment (zarov) [inline test/functional/GlobeControls.js:30]: Curious: is `cRL` breaking the test ?

File: utils/debug/Debug.js [hunk 2]
```diff
@@ -87,37 +93,43 @@ function Debug(view, datDebugTool, chartDivContainer) {
...
-                controls.addEventListener('camera-target-changed', cameraTargetListener);
-                listeners.push({ type: 'camera-target-changed', stateName: 'cameraTargetUpdated', fn: cameraTargetListener });
-            } else {
-                for (const listener of listeners) {
-                    controls.removeEventListener(listener.type, listener.fn);
-                    delete state[listener.stateName];
-                }
-                gui.removeFolder('Events');
+    // camera-target-updated event
+    let LatControler;
```
Comment (zarov) [inline utils/debug/Debug.js]: `controller` takes two `L` ;)

**Review by zarov — APPROVED** [review]

**Review by zarov — APPROVED** [review]
Looks good to me !

File: test/functional/GlobeControls.js (hunk 1 above)
Comment (gchoqueux) [reply test/functional/GlobeControls.js:30]: yes

File: utils/debug/Debug.js (hunk 2 above)
Comment (gchoqueux) [reply utils/debug/Debug.js]: fixed

**Pull request merged**